internal/pkg/cli/svc_init.go

```diff
@@ -564,6 +559,23 @@ func (o *initSvcOpts) askSvcPublishers() (err error) {
 	return nil
 }
 
+func validateInputApp(wsApp, inputApp string, store store) (string, error) {
```

Comment:
The string return is odd here, I'd expect `validateInputApp` to return only an error.
And then the client can set `o.appName = wsApp`